test/promise-component-mocha.js

```diff
@@ -118,6 +118,32 @@ describe('promise-component', function() {
             return onceData(component, 'outgoing', 'sentback', 'incoming', "hello");
         }).should.be.rejectedWith("hello world");
     });
+    it("should log rejections to console.error when no reject port attached", function() {
```

Comment:
The outport is created, but in this test nothing is attached to it (only the in/out ports are attached).